lib/index.d.ts

```diff
@@ -410,7 +419,8 @@ declare namespace Realm.Sync {
         ssl?: SSLConfiguration;
         error?: ErrorCallback;
         partial?: boolean;
-        _disablePartialSyncUrlChecks?:boolean;
+        fullSynchronization?: boolean;
```

Comment:
I don't understand why it's `full_synchronization` elsewhere and `fullSynchronization` here - is that a bug? I prefer the latter of the two.

Reply:
The JS name should be `fullSynchronization`, in C++ that turns into `full_synchronization`. If that isn't the case it is a bug.